quickstarts/maven/simple-dockerfile/Dockerfile

```
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+FROM openjdk:latest
+COPY maven/target/zero-config-dockerfile.jar /tmp/zero-config-dockerfile.jar
+CMD ["java", "-jar", "/tmp/zero-config-dockerfile.jar"]
```

Maybe the quickstart would be clearer if we provided a COPY statement with a file that exists in the project root directory in order to showcase how the inferred contextdirectory + assembly behaves.
```suggestion
COPY maven/target/zero-config-dockerfile.jar /tmp/zero-config-dockerfile.jar
COPY maven/static-dir-in-project-root/my-file.txt /tmp/my-file.txt
CMD ["java", "-jar", "/tmp/zero-config-dockerfile.jar"]
```